### PR TITLE
fix: fix for ci

### PR DESCRIPTION
**Pull request by likyh**
# Summary

fix some connection ci and skip pipeline ci

### CODE REVIEW

File: config-ui/cypress/integration/0-api/api.spec.js [hunk 1]
```diff
@@ -42,10 +42,8 @@ context('API Network Requests', () => {
         expect(response.body[0]).to.have.property('endpoint')
         expect(response.body[0]).to.have.property('username')
         expect(response.body[0]).to.have.property('password')
-        expect(response.body[0]).to.have.property('epicKeyField')
-        expect(response.body[0]).to.have.property('storyPointField')
-        expect(response.body[0]).to.have.property('remotelinkCommitShaPattern')
         expect(response.body[0]).to.have.property('proxy')
+        expect(response.body[0]).to.have.property('rateLimitPerHour')
```
Comment (e2corporation) [inline config-ui/cypress/integration/0-api/api.spec.js:46]: The response key from the API is actually just `rateLimit`. All property assertions for rate limit should be updated to `rateLimit`.

**Review by e2corporation — APPROVED** [review]
@likyh Nice work, one small fix needed (`rateLimit` key) and rebase then LGTM.

File: config-ui/cypress/integration/0-api/api.spec.js (hunk 1 above)
Comment (likyh) [reply config-ui/cypress/integration/0-api/api.spec.js:46]: image
The response is rateLimitPerHour. Is it changed in any unmerged PR?

Comment (e2corporation) [reply config-ui/cypress/integration/0-api/api.spec.js:46]: @likyh After rebuilding `main` it looks like it has been changed to `rateLimitPerHour` on response as well. No change necessary in that case.

**Pull request merged**